### ISSUE TITLE
Add JS methods to get current `center` and `zoom` values

### ISSUE THREAD

**Issue by lindseydiloreto** [doublesecretagency/craft-googlemaps#35]
If a front-end map has been moved or zoomed by the user, we may need to know the **current** center coordinates and zoom level. It's already possible to get this information, but it's a little ugly...

```js
let map = googleMaps.getMap('MAPID');
let center = map._map.center;
let zoom   = map._map.zoom;
```

We should add a couple of methods to make this a little more intuitive...

```js
let map = googleMaps.getMap('MAPID');
let center = map.getCenter();
let zoom   = map.getZoom();
```

**Comment by lindseydiloreto** [comment]
Similarly, we should elevate `getBounds`...

```js
// Wrapper for `map._map.getBounds()`
let bounds = map.getBounds();
```

**Comment by lindseydiloreto** [comment]
This is done, and will be available in **v4.1.4**
